### PR TITLE
Feature/infolist item playground

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes 3489 .

#### Changes proposed in this Pull Request:
- Playground for InfolistItem
- 
- 

#### Screenshots:
Screenshot 2022-09-12 at 4 28 27 PM

#### To Test:
- http://localhost:3000/components/info-list-item/playground

### CODE REVIEW

**Comment by emclaug2** [comment]
It might be beneficial to add some sort of light border around the InfoListItem, or give it a background color by defaut so we can see where it starts & ends.

**Comment by emclaug2** [comment]
Code generation looking good; one thing I've noticed - can you remove the `info` & `subtitle` props when no value is provided?

image

**Comment by jeffvg** [comment]
should the top accordion in playground have a title? other than that looks good

image

**Comment by emclaug2** [comment]
@ektasawant The changes look good; I've got the same question as Jeff:

image


Also one more thing, a style suggestion - I've added a max-width of 700px to the InfoListItem example since it looks a little weird if it stretches all the way across the screen.

https://blui-angular-docs--pr495-feature-3442-infolis-2blc8cfa.web.app/components/info-list-item/playground

**Comment by ektasawant** [comment]
> https://blui-angular-docs--pr495-feature-3442-infolis-2blc8cfa.web.app/components/info-list-item/playground

- Added `required props` as a section name.
- Added max-width 700px

**Review by jeffvg — APPROVED** [review]
looks good

File: docs/src/componentDocs/InfoListItem/playground/InfoListItemConfig.tsx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+            propType: 'string',
+            helperText: 'Color override for the chevron icon',
+            required: false,
+        },
+        {
+            propName: 'dense',
+            inputType: 'boolean',
+            inputValue: false,
+            propType: 'boolean',
+            helperText: 'Smaller height row with less padding	',
```
Comment (daileytj) [inline docs/src/componentDocs/InfoListItem/playground/InfoListItemConfig.tsx:46]:
```suggestion
            helperText: 'Smaller height row with less padding',
```

**Review by daileytj — APPROVED** [review]
Looks good to me

**Comment by daileytj** [comment]
Screen Shot 2022-09-13 at 9 21 28 AM
 WrapTitle, WrapSubtitle, and WrapInfo aren't working as expected. I thought this may be an issue with the component, but they work fine within the old storybook so I don't think this is a component issue.

**Comment by ektasawant** [comment]
> https://blui-react-docs--pr512-feature-infolistitem-ai73aaik.web.app

Screenshot 2022-09-13 at 8 15 44 PM
I am getting this when checked WrapTitle, WrapSubtitle, and WrapInfo.

**Comment by daileytj** [comment]
@ektasawant Looks fine. Must have been an issue with my local build. ¯\_(ツ)_/¯

**Pull request merged**